modules/core/src/smithy4s/schema/Alt.scala

```diff
@@ -86,51 +101,48 @@ object Alt {
         encoderK: EncoderK[G, Result]
     ): G[U]
 
-    def projector[A](alt: Alt[U, A]): U => Option[A]
+    def ordinal(u: U): Int
```

Comment:
**thought:** could we make it more constrained than just `Int`, in dispatchers? maybe a type member?

then, we'd also need methods like `def get[A](i: Ordinal)(items: Seq[A]): A`.

Reply:
We could have `get[A](u: U)(items: Seq[A]): A` ?

Reply:
that... works. but now I'm afraid of people passing a seq of the wrong length. It's equally bad :( 

the whole idea is reminiscent of `Representable` from cats